src/main.cpp

```diff
@@ -78,10 +92,56 @@ int main(int argc, char* argv[]) {
     novelChanRect->setRotation(rotation);
   });
 
-
-  auto rect = runner.getInteractionService()->getBasicInteractionRect(NovelRT::GeoVector<float>(200, 200), rectArgs);
-  rect->subscribeToInteracted([]{novelChanRect->setActive(!novelChanRect->getActive());});
-
+  auto novelAudio = runner.getAudioService();
+
+  novelAudio->loadMusic("sparta.wav");
+  novelAudio->loadSound("w0nd0ws.wav");
+  novelAudio->fadeMusicIn("sparta.wav", -1, 5000);
+  novelAudio->setGlobalVolume(0.5);
+
+
+  auto rect = runner.getInteractionService()->getBasicInteractionRect(NovelRT::GeoVector<float>(200, 200), rectArgs2);
+
+  rect->subscribeToInteracted([&novelAudio]{
+      std::cout << "Commencing SFX test..." << std::endl;
+      novelAudio->fadeMusicOut(500);
+      SDL_Delay(600);
```

Comment:
Isn't this going to prevent the render loop from running >.>

Reply:
Yes, its only for demonstration as of now. Ruby and I had discussed this on Discord - most likely we'll need to handle this, but it would be part of the HLAPI and separate from this PR (unless you have some input that would go against that?)

Reply:
Ah...yeah this should probably be cleaned up as per #13 .

Reply:
To clarify - just make this not block the render loop for now, the HLAPI stuff will be a different PR.

Reply:
Erm, that will make every channel play on top of itself which will not work - in that case, should I just remove this demo?

Reply:
In this case, you could pretty trivially not block the render loop by just checking the elapsed time since the last call was played.

Reply:
I removed this test and replaced with an interactive one - I'd like to re-implement at a later time.